vlc-ios/AppUtitlity.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  AppUtitlity.swift
+//  VLC-iOS
+//
+//  Created by Carola Nitz on 10.07.19.
+//  Copyright © 2019 VideoLAN. All rights reserved.
+//
```

Comment:
I believe that the header style should be the one we generally use in this project.

Reply:
oh yeah Thanks! I'll update